python/tests/test_frames.py

```diff
@@ -162,3 +166,46 @@ def test_wrong_linkage(assert_log):
 
     f.link([f])
     assert_log("wrong linkage")
+
+def test_frame_index():
+    frame = makeframe()
+    frame.index_type = 'DECREASING'
+
+    assert frame.index == frame.channels[0]
```

Comment:
I think that test would be more clear if you were able to verify frame.index == time0 without refering to channels[0]. Otherwise it looks way to much like you are just repeating implementation.